examples/src/test/scala/org/virtuslab/ideprobe/ModuleTest.scala

```diff
@@ -2,16 +2,56 @@ package org.virtuslab.ideprobe
 
 import java.nio.file.Files
 
-import org.junit.Assert
+import org.junit.{Assert, Test}
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.ValueSource
 import org.virtuslab.ideprobe.Extensions._
-import org.virtuslab.ideprobe.protocol.{ModuleRef, Setting, TestRunConfiguration}
+import org.virtuslab.ideprobe.protocol._
 import org.virtuslab.ideprobe.scala.ScalaPluginExtension
 import org.virtuslab.ideprobe.scala.protocol.SbtProjectSettingsChangeRequest
 
-
 class ModuleTest extends IdeProbeFixture with ScalaPluginExtension {
+  @Test
+  def runJUnitTestsInDifferentScopes: Unit = fixtureFromConfig("projects/dokka.conf").run { intelliJ =>
+    deleteIdeaSettings(intelliJ)
+    intelliJ.probe.openProject(intelliJ.workspace)
+
+    val moduleName = intelliJ.config[String]("test.module")
+    val className = intelliJ.config[String]("test.class")
+    val methodName = intelliJ.config[String]("test.method")
+    val moduleRef = ModuleRef(moduleName)
+
+    val runConfigurations = List(
+      JUnitRunConfiguration.module(moduleRef),
+      JUnitRunConfiguration.mainClass(moduleRef, className),
+      JUnitRunConfiguration.method(moduleRef, className, methodName)
+    )
+
+    runConfigurations.map(intelliJ.probe.run).foreach { result =>
+      Assert.assertTrue(s"Test result $result should not be empty", result.suites.nonEmpty)
+    }
+  }
+
+  @Test
+  def runScalaTestTestsInDifferentScopes: Unit = fixtureFromConfig("projects/io.conf").run { intelliJ =>
```

Comment:
does ci run this?

Reply:
no, do we even want examples to be run on ci? They depend on repositories that can change in the future.